cypress/integration/login.js

```diff
@@ -1,23 +1,42 @@
-describe('Sourcebooks login', function() {
+import LoginPage from '../objects/loginPage';
+import TimeLoggingPage from '../objects/timeLoggingPage';
+
+const loginPage = new LoginPage();
+const timeLoggingPage = new TimeLoggingPage();
 
+function prepareSUT () {

```

Comment:
will this method be reused? Because now i think more simple way would be to put all the variable declarations in `describe `block just before `it`

Reply:
I don't know the scope of this workshop, so if we are going to implement more tests, preparation of test suite does make sense, therefore I'd rather choose to have it.

Reply:
I think I second what David said, but if you really think you need it, it does work so I reckon there ain't no harm in it.